compiler/ballerina-lang/src/main/resources/grammar/BallerinaParser.g4

```diff
@@ -109,7 +110,7 @@ sealedLiteral
 restDescriptorPredicate : {_input.get(_input.index() -1).getType() != WS}? ;
 
 objectFunctionDefinition
-    :   documentationString? annotationAttachment* deprecatedAttachment? (PUBLIC | PRIVATE)? (REMOTE | RESOURCE)? EXTERN? FUNCTION callableUnitSignature (callableUnitBody | SEMICOLON)
+    :   documentationString? annotationAttachment* deprecatedAttachment? (PUBLIC | PRIVATE)? (REMOTE | RESOURCE)? FUNCTION callableUnitSignature (callableUnitBody | (ASSIGN EXTERNAL)? SEMICOLON)
```

Comment:
This allows having an external resource method, which is not supported by the runtime. I have created issue  https://github.com/ballerina-platform/ballerina-spec/issues/32 to capture this in the spec. Shall we disable in the compiler, saying this implementation doesn't support this functionality?

Reply:
Disallowed external resource functions with e8b0cbb, 371e4b0.